src/index.ts

```diff
@@ -607,19 +609,15 @@ export class Bigtable {
   /**
    * @typedef {array} GetInstancesResponse
    * @property {Instance[]} 0 Array of {@link Instance} instances.
-   * @property {object} 1 The full API response.
-   *     Note: 'failedLocations' property may contain locations from which
-   *     Instance information could not be retrieved.
-   *     Values are of the form `projects/<project>/locations/<zone_id>`
+   * @property {string[]} 1 locations from which Instance information could not be retrieved
+   * @property {object} 2 The full API response.
    */
   /**
    * @callback GetInstancesCallback
    * @param {?Error} err Request error, if any.
    * @param {Instance[]} instances Array of {@link Instance} instances.
+   * @param {string[]} locations from which Instance information could not be retrieved
    * @param {object} apiResponse The full API response.
-   *     Note: 'failedLocations' property may contain locations from which
```

Comment:
@bcoe wdyt about keeping these comments?

Reply:
Do we know whether `failedlocations` is actually populated? I'm curious about the motivation for its removal.

If it's populated, I see no reason not to have the comments?